websites/H/HDrezka/presence.ts

```diff
@@ -0,0 +1,57 @@
+const presence = new Presence({
+	clientId: "1191450515670843533",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			details: "Где-то на сайте",
+			largeImageKey:
+				"https://cdn.discordapp.com/app-assets/1191450515670843533/1247862870276833292.png",
```

Comment:
Replace these with `https://i.imgur.com/0gTpbcv.png` which is a 512x512 image

Reply:
done